docs/Helm.md

```diff
@@ -0,0 +1,106 @@
+# Splunk Operator Helm Installation
+
+## Splunk Operator Helm chart Repository
+
+Add the Splunk Operator and Enterprise charts to your Helm repository.
+
+```
+helm repo add splunk https://splunk.github.io/splunk-operator/
+helm repo update
+```
+
+The ```splunk``` chart repository contains the ```splunk/splunk-operator``` chart to deploy the Operator and the ```splunk/splunk-enterprise``` chart to deploy Splunk Custom Resources.
+
+Helm provides a long list of commands to manage your deployment, we'll be going over a few useful ones in the sections to come. You can learn more about supported commands [here](https://helm.sh/docs/helm/helm/).
+
+## Splunk Operator deployments
+
+To install the Splunk Operator:
+
+```
+helm install <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+To upgrade the operator, specify override configurations through the CLI or a new file:
+
+```
+helm upgrade --set splunkOperator.clusterWideAccess=false <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Here we've upgraded the operator by revoking cluster-wide access.
+
+To rollback the operator to a previous release:
+
+```
+helm rollback <RELEASE_NAME> <REVISION_NUMBER> -n <RELEASE_NAMESPACE>
```

Comment:
Though this is a very good option at the Helm level, i think it can cause issues, when it comes to the Splunk deployment. I am not sure if this is even supported, particularly with the major releases, as we may not have reverse migration logic in place . Also,  i guess the order of the migration is also important, even if it is some minor releases.

Reply:
I agree, I've removed the section for now.